### PR TITLE
Update our step for 4-15 branching

### PR DESCRIPTION
**Pull request by Sandeepyadav93**
https://issues.redhat.com/browse/CNF-10592

### CODE REVIEW

**Review by sshnaidm — COMMENTED** [review]
thanks, one nit

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 1]
```diff
@@ -250,9 +263,12 @@ fi
 export CNF_E2E_TESTS
 export CNF_ORIGIN_TESTS
 
-if [[ "$T5CI_VERSION" == "4.15" ]]; then
+if [[ "$T5CI_VERSION" == "4.16" ]]; then
     export CNF_BRANCH="master"
     export CNF_TESTS_IMAGE="cnf-tests:4.14"
+elif [[ "$T5CI_VERSION" == "4.15" ]]; then
+    export CNF_BRANCH="release-${T5CI_VERSION}"
+    export CNF_TESTS_IMAGE="cnf-tests:4.14"
```
[the remark this answers is not in the document]
Comment (Sandeepyadav93) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:271]: @sshnaidm ack, just to confirm do you mean something like this?

~~~
if [[ "$T5CI_VERSION" == "4.16" ]]; then
    export CNF_BRANCH="master"
else
    export CNF_BRANCH="release-${T5CI_VERSION}"
fi


if [[ "$T5CI_VERSION" == "4.15" ]] || [[ "$T5CI_VERSION" == "4.16" ]]; then
    export CNF_TESTS_IMAGE="cnf-tests:4.14"
else
    export CNF_TESTS_IMAGE="cnf-tests:${T5CI_VERSION}"
~~~

[the remark this answers is not in the document]
Comment (Sandeepyadav93) [reply ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:274]: done

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse ack


4.16 results looks good: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/46536/rehearse-46536-periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-cnftests/1734777448744095744/artifacts/e2e-telco5g-cnftests/telco5g-cnf-tests/artifacts/test_results.html

**Pull request merged**